config/settings.yaml.example

```diff
@@ -54,3 +54,7 @@
 #    :enabled: false
 #  :templates:
 #    :enabled: true
+#  :action:
+#    :enabled: true
```

Comment:
I find the name action unfriendly, not sure why would users ever want to look in the "action" logs?